i18n/translation-en.json

```diff
@@ -307,7 +307,9 @@
     "pan": "Pan",
     "select": "Select",
     "Marquee": "Marquee Zoom",
-    "Link": "Link URL or Page"
+    "Link": "Link URL or Page",
+    "StandardBusiness": "Standard Business",
```

Comment:
When you have time can you follow the wiki https://github.com/XodoDocs/webviewer/wiki/WebViewer-UI-Localization to translate them into other languages? Thanks.